landoui/templates/queue/queue.html

```diff
@@ -0,0 +1,82 @@
+{% extends "partials/layout.html" %}
+{# Fake queue data till the API is ready #}
+{% 
+  set queue_data = [
+    { 
+      'priority': 1,
+      'requester_email': "user@example.com",
+      'bug_ids': [123456, 123457, 123458],
+      'repo_name': 'mozilla-central',
+      'revision_order': [1001, 1002, 1003],
+      'created_at': '2019-10-08T06:24:40.208168+00:00',
+    },
+    { 
+      'priority': 2,
+      'requester_email': "imadueme@mozilla.com",
+      'bug_ids': [323456],
+      'repo_name': 'mozilla-central',
+      'revision_order': [1504],
+      'created_at': '2019-10-08T06:24:53.10194+00:00'
+    },
+  ]
+%}
+{% block main %}
+<main class="QueuePage container">
+  <h1>Landing Queue</h1>
+  <div class="QueuePage-options">
+    <div class="QueuePage-optionsRepo">
+      <span>Respository: </span>
+      <select>
+          <option>
+            Mozilla-Central
+          </option>
+      </select>
+    </div>
+    
+    <div class="QueuePage-optionsIsMine">
+      <span>Show:</span> &nbsp;
+      <a href="#" data-show="all">All</a> &nbsp; | &nbsp;
+      <a href="#" data-show="just-mine">Just Mine</a>
+    </div>
+  </div>
+
+  <div class="QueuePage-listHeaders">
+    <div class="QueuePage-listHeadersPriority">Priority</div>
```

Comment:
The way this is currently mocked means this is more of "position" than priority. priority will be equal for all rows, except special ones which are bumped. Although the UI doesn't need to match the underlying data model, I think position just might be more clear in general, thoughts?

Reply:
Actually that makes sense, priority doesn't determine the position in the list exactly. I'll update it.